stubs/pywin32/isapi/threaded_extension.pyi

```diff
@@ -1,26 +1,28 @@
 import threading
 from _typeshed import Incomplete
+from collections.abc import Callable
+from typing import Final, Literal
 
 import isapi.simple
 
-ISAPI_REQUEST: int
-ISAPI_SHUTDOWN: int
+ISAPI_REQUEST: Final = 1
+ISAPI_SHUTDOWN: Final = 2
 
 class WorkerThread(threading.Thread):
     running: bool
-    io_req_port: Incomplete
+    io_req_port: int
     extension: Incomplete
-    def __init__(self, extension, io_req_port) -> None: ...
+    def __init__(self, extension, io_req_port: int) -> None: ...
     def call_handler(self, cblock) -> None: ...
 
 class ThreadPoolExtension(isapi.simple.SimpleExtension):
     max_workers: int
     worker_shutdown_wait: int
-    workers: Incomplete
-    dispatch_map: Incomplete
-    io_req_port: Incomplete
+    workers: list[WorkerThread]
+    dispatch_map: dict[int, Callable[..., None]]
```

Comment:
`object` to say that the return value is unused, rather than forcing it to be `None`.
And we have an alias for that: `_typeshed.Unused`.

```suggestion
    dispatch_map: dict[int, Callable[..., Unused]]
```

![image](https://github.com/user-attachments/assets/2e3a3c1a-9272-4d50-85ae-c1503ed5171d)
![image](https://github.com/user-attachments/assets/2b328911-bcb3-4f13-8e7d-c07e925b361d)